### ISSUE TITLE
New "music" folder in mod directory

### ISSUE THREAD

**Issue by gigapouch** [RigsOfRods/rigs-of-rods#291]
A nice feature would be to be able to play custom music in the game, like ETS2. The music could be located in the RoR 0.4 folder in the documents.

**Comment by Hiradur** [comment]
IMO it makes more sense to have a music player running in the background because otherwise you would have to copy your music to RoR, creating redundancy. Symlinking could reduce redundancy but it's still somewhat inconvenient.

**Comment by gigapouch** [comment]
OK, maybe not a folder, but a built in RoR music player where you can select a folder with music.

**Comment by Hiradur** [comment]
That would be better but is it really worth it? There are very good music players out there and running them in the background shouldn't do much harm. The only benefit would be that you don't have to alt-tab out of them game to get to the player. 
Some players provide overlays for games though so they basically act like an ingame player with the added benefit that they are already stable software and they spare us the time to code our own player.
Other opinions?

**Comment by gigapouch** [comment]
I didn't know you could do Alt-tab.

**Comment by TheUnknownGuyFromRoR** [comment]
you can also run RoR in windowed mode. won't loose anything, but will be able to run everything else you want

**Comment by gigapouch** [comment]
But you can't interact with the OS in Windowed mode, though.

**Comment by DirtGamer301** [comment]
If I do anything to get to Windows and then go back to RoR, no matter if windowed mode or not, the cars textures will be screwed like the example picture, so this wouldn't work for all people.
+Here's another disadvantage I found:
How to reproduce: Run RoR in full screen mode, open a YouTube playlist (not tried in any other way):
The playlist will stop after each video, and keep going if you turn RoR off. I haven't tried it for other websites (music only websites for example), other playlists or windowed mode or things as I never really cared and it was just something random I noticed.

What would probably work is teaming up with a music website (for free music wich is free to use, I can't remember a name atm though) to choose a playlist from the website and play it ingame.
